mypy.ini

```diff
@@ -0,0 +1,5 @@
+[mypy]
+ignore_missing_imports = True
```

Comment:
Do we want this to be true? Is there a particular case where this causes some kind of false positive?

Reply:
I tried listing external libraries one-by-one when they failed mypy for not good reasons. Gave up eventually.